Cabal/doc/conf.py

```diff
@@ -15,7 +15,7 @@
 
 extensions = ['sphinx.ext.extlinks']
 
-templates_path = ['.templates']
+templates_path = ['_templates']
```

Comment:
That is a source code folder and not temp/config folder. And naming it `_templates` is standard for sphinx docs.